src/components/Match/MatchLog.jsx

```diff
@@ -89,6 +118,11 @@ class MatchLog extends React.Component {
         <Checkbox defaultChecked label={checkbox.label} onCheck={(event, isChecked) => this.handleCheck(checkbox.label, isChecked)} />
       </span>))}
       <Table data={generateLog(this.props.match, this.state)} columns={logColumns} />
+      <div>{[...Array(7)].map((_, n) => {
+        const runeString = strings[`rune_${n}`];
+        return <ReactTooltip id={runeString}><span>{runeString}</span></ReactTooltip>;
```

Comment:
This adds tooltips, together with the data-tip and data-for attributes added to the image tag. It is a bit fragile because we depend on there being seven runes. However, the most straightforward alternative was to make a separate tooltip object for each rune event.

A more sophisticated alternative would be to build a list of runes actually used on the page in `generateLog`. And in fact this could also be done for the heroes, which currently have no tooltips.